qualification/report/generate_pdf.py

```diff
@@ -1050,7 +1058,11 @@ def _doc_result(section: Container, result: Result, tmp_dir: pathlib.Path, figur
                     procedure_table.add_row((MultiColumn(2, align="|c|", data=mp),))
                 procedure_table.add_hline()
 
-    if result.failure_messages:
+    if result.cbf is not None and result.cbf.error is not None:
+        # In this case we'll already have details about the failure as part
+        # under the heading for the CBF, so don't repeat the whole thing here.
+        section.append(TextColor("red", f"CBF failed to start: {result.cbf.error.splitlines()[-1]}"))
```

Comment:
> [...] we'll already have details about the failure **as part under the heading** for the CBF, [...]

"as part under the heading"? 
* Typo? Shorthand? Not quite understanding the comment.